lstchain/irf/hdu_table.py

```diff
@@ -0,0 +1,374 @@
+    deltaT = deltaT[(deltaT > 0) & (deltaT < 0.002)]
+    rate = 1 / np.mean(deltaT)
+    dead_time = np.amin(deltaT)
+
+    # Position parameters
+    location = EarthLocation.from_geodetic(
+        -17.89139 * u.deg, 28.76139 * u.deg, 2184 * u.m
+    )
+    reco_alt = data["reco_alt"]
+    reco_az = data["reco_az"]
+    pointing_alt = data["pointing_alt"]
+    pointing_az = data["pointing_az"]
+
+    src_sky_pos = SkyCoord(
+        alt=reco_alt, az=reco_az, frame=AltAz(obstime=time, location=location)
+    ).transform_to(frame="icrs")
+    tel_pnt_sky_pos = SkyCoord(
+        alt=pointing_alt[0],
+        az=pointing_az[0],
+        frame=AltAz(obstime=time[0], location=location),
+    ).transform_to(frame="icrs")
+
+    try:
+        object_radec = SkyCoord.from_name(source_name)
+    except Exception:
+        log.error("Name resolve error in finding the Object in Sesame")
+        object_radec = SkyCoord(tel_pnt_sky_pos.icrs)
+
+    # Observation modes
+    source_pointing_diff = object_radec.separation(
+        SkyCoord(tel_pnt_sky_pos.ra, tel_pnt_sky_pos.dec)
+    ).deg
+    # Assuming wobble offset is fixed to 0.4
+    if round(source_pointing_diff, 1) == 0.4:
+        mode = "WOBBLE"
+    elif round(source_pointing_diff, 1) > 1:
+        mode = "OFF"
+    elif round(source_pointing_diff, 1) == 0.0:
+        mode = "ON"
+    else:
+        # Nomenclature is to be worked out or have a separate way to mark mispointings
+        mode = "UNDETERMINED"
+
+    log.error(
+        f"Source pointing difference with camera pointing is {source_pointing_diff:.3f} deg"
+    )
+
+    event_table = QTable(
+        {
+            "EVENT_ID": u.Quantity(data["event_id"]),
+            "TIME": u.Quantity(data["dragon_time"]),
+            "RA": u.Quantity(src_sky_pos.ra.to(u.deg)),
+            "DEC": u.Quantity(src_sky_pos.dec.to(u.deg)),
+            "ENERGY": u.Quantity(data["reco_energy"]),
+        }
+    )
+    gti_table = QTable(
+        {"START": u.Quantity(t_start, ndmin=1), "STOP": u.Quantity(t_stop, ndmin=1)}
+    )
+    pnt_table = QTable()
+
+    # Adding the meta data
+    # Event table metadata
+    ev_header = DEFAULT_HEADER.copy()
+    ev_header["HDUCLAS1"] = "EVENTS"
+
+    ev_header["OBS_ID"] = run_number
+    ev_header["DATE_OBS"] = date_obs
+    ev_header["TSTART"] = t_start
+    ev_header["TSTOP"] = t_stop
+    ev_header["MJDREFI"] = "40587"  # mjd format for date 01-01-1970
+    ev_header["MJDREFF"] = "0"
+    ev_header["TIMEUNIT"] = "s"
+    ev_header["TIMESYS"] = "UTC"
+    ev_header["OBJECT"] = source_name
+    ev_header["OBS_MODE"] = mode
+    ev_header["N_TELS"] = len(tel_list)
+    ev_header["TELLIST"] = "LST-" + f" ".join(map(str, tel_list))
+
+    ev_header["RA_PNT"] = tel_pnt_sky_pos.ra.value
+    ev_header["DEC_PNT"] = tel_pnt_sky_pos.dec.value
+    ev_header["ALT_PNT"] = data["pointing_alt"].mean().to_value(u.deg)
+    ev_header["AZ_PNT"] = data["pointing_az"][0].to_value(u.deg)
```

Comment:
I remember this was already asked by @maxnoe. I still see that one uses mean and another just the first element. Is it fine like this?

Reply:
Yes, this is just a matter of convenience. `pointing_az` has very close values, so I avoid using mean

Reply:
Please use the mean. That will be applicable (or sensical) for all runs, not just the one you looked at that didn't change much over the run. But in general, e.g. when tracking sources close to the celestial equator, azimuth will vary more strongly than zenith during a run.

Reply:
Understood. Changed accordingly